batch/batch/front_end/query/query.py

```diff
@@ -280,14 +281,14 @@ def query(self) -> Tuple[str, List[Any]]:
         return (sql, [self.time_msecs])
 
 
-class DurationQuery(Query):
+class JobDurationQuery(Query):
     @staticmethod
-    def parse(op: str, time: str) -> 'DurationQuery':
+    def parse(op: str, time: str) -> 'JobDurationQuery':
         operator = get_operator(op)
         if not isinstance(operator, ComparisonOperator):
             raise QueryError(f'unexpected operator "{op}" expected one of {ComparisonOperator.symbols}')
-        time_msecs = int(parse_float(time) * 1000 + 1)
-        return DurationQuery(operator, time_msecs)
```

Comment:
What was this + 1 for and why is it removed?

Reply:
I wasn't sure which direction the value needed to be rounded. When I looked at the generated SQL statement for this query `duration > 5000`, then I got `((time_completed - time_started) > 5000001` (converting to msecs) which is not what I think we want. Maybe we can just keep the float comparison and MySQL will do the numeric comparison appropriately?

Reply:
Hm, I think what sticks out to me a little more is that we can't really guarantee users precision beyond the millisecond. If I query for `duration > 0.0015` and the job is recorded as 1ms, should I return that value or not? I don't think we can really know because the real time of the job *could* have been 1.9ms or it could have been 1.2ms. I think it would be best just to assume that these floats map to whole milliseconds (`int(duration * 1000)`) and maybe as a later project we can write a little client-side javascript that looks at the search box and shows a warning if a user tries to get finer precision than we can promise (I imagine this will basically never happen).